lua/wikis/commons/Infobox/Extension/PatchAuto.lua

```diff
@@ -47,31 +47,23 @@ function PatchAuto._fetchPatchData(patch, patchDisplay)
 		return {link = patch, display = patchDisplay}
 	end
 
-	patch = patch:gsub(' ', '_')
-
-	local patchData = mw.ext.LiquipediaDB.lpdb('datapoint', {
-		conditions = '[[type::patch]] AND [[pagename::' .. patch .. ']]',
-		query = 'name',
+	local patchData = Patch.queryPatches{
+		additionalConditions = ConditionNode(ColumnName('pagename'), Comparator.eq, patch:gsub(' ', '_')),
```

Comment:
Is it correct to query this by pagename? I would have assumed that the patch input is a version? or am I wrong here?

Reply:
Let's update SG Tournament Pages, or?